packages/mui-joy/src/CssBaseline/CssBaseline.tsx

```diff
@@ -54,6 +54,12 @@ function CssBaseline(props: CssBaselineProps) {
             'strong, b': {
               fontWeight: theme.vars.fontWeight.lg,
             },
+            a: {
+              color:
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primary[300]
```

Comment:
```suggestion
                  ? (theme.vars || theme).palette.primary[300]
```

Reply:
@brijeshb42 would an approach similar to what I pushed for the Material version work? It's following Marija's guidance and it definitely looks cleaner not having to call the dark mode ternary here 😬